src/com/xilinx/rapidwright/edif/EDIFCell.java

```diff
@@ -245,41 +240,45 @@ public EDIFPort addPort(EDIFPort port) {
         port.setParentCell(this);
         EDIFPort collision = ports.put(port.getBusName(), port);
         if (collision != null && port != collision) {
```

Comment:
Assuming that `port` can't be `null`, I think this is sufficient:
```suggestion
        if (port != collision) {
```